opensource.html

```diff
@@ -52,7 +52,13 @@ <h2>Open Source Implementations of OpenC2 on other GitHub repos</h2>
         <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/StephenOTT/OpenC2-Worker">OpenC2 polymorphic event-driven non-blocking Worker</a></li>
         <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/StephenOTT/OpenC2-Orchestrator">OpenC2 Orchestrator backed by BPMN</a></li>
         <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/TedDriggs/openc2-rs">OpenC2 types for Rust</a></li>
+        <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/sFractal-Podii/quizquadaminos">QuadBlockQuiz - a game which includes OpenC2 questions and website is OpenC2 managed</a></li>
+        <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/sparrell/BlinkyHaHa">BlinkyHaHa - Blinky Https Api Helloworld AP</a></li>
+        <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/sparrell/BlinkyMaHa">BlinkyMaHa - Blinky Mqtt Api Helloworld AP - MQTT version of BlinkyHaHa</a></li>
+        <li><a rel="noopener noreferrer" target="_blank" href="https://github.com/sFractal-Podii/TwinklyMaHa">TwinklyMaHa - Twinkly is the digital twin of blinky (ie in cloud instead of on Raspberry Pi, LiveView graphics instead of LEDs). MQTT API</a></li>

```

Comment:
grammar nits, apply to lines 58 and 59:  

- "ie" is properly written "i.e.,"  
- Should "blinky" be capitalized?